layout-service/src/main/java/io/meeds/layout/plugin/attachment/CmsPortletAttachmentPlugin.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2020 - 2024 Meeds Association contact@meeds.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package io.meeds.layout.plugin.attachment;
+
+import io.meeds.layout.service.LayoutAclService;
+import io.meeds.layout.service.PortletInstanceService;
+import jakarta.annotation.PostConstruct;
+import org.exoplatform.commons.exception.ObjectNotFoundException;
+import org.exoplatform.services.security.Identity;
+import org.exoplatform.social.attachment.AttachmentPlugin;
+import org.exoplatform.social.attachment.AttachmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class CmsPortletAttachmentPlugin extends AttachmentPlugin {
```

Comment:
Shouldn't we use the same Plugin and Object Type than `Image` portlet ?

Reply:
For new created portlet (loginForm, sidebar, platformName, platformLogo), we need 
- an AttachmentPlugin
- a TranslationPlugin

Current ImageAttachmentPlugin is in content module, and there is no translationPlugin for "portlet" so we need to create one.

Using ImageAttachmentPlugin means that we use one objectType for attachments ('imagePortlet'), and one for translations ('cmsPortlet'). 
I find it is clearer to use the same objectType ('cmsPortlet') for attachment AND translations for theses portlets.
That's why I create CMSPortletAttachmentPlugin.